src/lib/Ouroboros/Byron/Proxy/Main.hs

```diff
@@ -798,9 +823,8 @@ withByronProxy trace bpc idx db mempool k = do
               (Proxy :: Proxy MCVssCertificate)
               (\(MCVssCertificate vc) -> getCertId vc)
 
-          -- Nothing to do for delegation certificates.
-          -- FIXME verify that this meets requirements.
-          , postPskHeavy = \_ -> pure True
+          -- Do not put the cert into the mempool. Do not relay it (give False).
+          , postPskHeavy = \_ -> pure False
```

Comment:
Before it gave `True`, which would cause the Byron relay system to send the certificate to its peers. But it was determined that the proxy should _not_ relay these.